File: ejemplosClase/src/ejemplosClase/operacionesMatematicas.java
package ejemplosClase;

public class operacionesMatematicas {

	public static void main(String[] args) {
		int a = 5;
		int b = 70;
		int c = 13;
		
		int d=14,e=15;
		
		System.out.printf("a+b= %s\n",a+b);
		
		System.out.printf("c/b = %5.3f\n",(double)c/b);
		//lo mismo pero guardando la informacion
		double division = (double)c/a;
		System.out.printf("c/a = %s\n",division);
		
		System.out.printf("d %% a = %s\n",d%a);
		
		double div = (double)c/b;
		
		System.out.printf("Floor: %s\n", Math.floor(div));
		System.out.printf("Ceil: %s\n", Math.ceil(div));
		System.out.printf("Round: %s\n", Math.round(div));
		
		System.out.printf("Maximo: %s\n",Math.max(a, Math.max(b, c)));
		
		
		
		
		
		
		
		
		
		
		
		
		
		

	}

}

File: ejemplosClase/python/OperacionesMatematicas.py
#Required imports (math package)
import math

# Declaramos e initializamos algunas variables numericas
a = 5
b = 70
c = 13
d = 14
e = 15




		
#Suma de dos valores
print("a+b=",a+b)

# Division de dos valores: importante hacer el 'cast' 
print("c/b = ",c/b)

#lo mismo pero guardando la informacion
division = c/a
print("c/a = %s",division)

# Calcular el modulo
print("d % a = ",d%a)

# Rondear variables
div = float(c/b)
print("Floor:", math.floor(div))
print("Ceil:",math.ceil(div))
print("Round:", round(div))


# Calcular el maximo de a,b y c
print("Maximo: ",max(a, max(b, c)))

